setup.py

```diff
@@ -243,6 +243,8 @@ def _find_include_dir(self, dirname, include):
 
 
 def _cmd_exists(cmd):
+    if "PATH" not in os.environ:
+        return
```

Comment:
Let's be explicit about the return type (we use it in an `if`), and add type hints as well.

```suggestion
def _cmd_exists(cmd: str) -> bool:
    if "PATH" not in os.environ:
        return False
```